src/pages/tutor-home/TutorHome.tsx

```diff
@@ -37,6 +38,7 @@ const TutorHome = () => {
         description={t('tutorHomePage.popularCategories.description')}
         title={t('tutorHomePage.popularCategories.title')}
       />
+      <TutorSchedule />
```

Comment:
`TutorSchedule` block should be below the  `FindBlock`, according to [figma](https://www.figma.com/design/liJZDYhUnDP15Pi9bipDcv/Space2Study-(Students)?node-id=2140-54676&t=hIwawP1kVQcUFyJK-4)